pytests/tests/test_awaitable.py

```diff
@@ -1,13 +1,22 @@
 import pytest
+import sys
 
 from pyo3_pytests.awaitable import IterAwaitable, FutureAwaitable
 
 
+@pytest.mark.skipif(
+    sys.implementation.name == "graalpy",
+    reason="GraalPy's asyncio module does not work with native classes",
```

Comment:
Is there an issue for this we can link to, or will never work?

Reply:
We have a bug, I only noticed it when rebasing, since this test didn't exist before. I can open a Github issue for it against oracle/graalpython

Reply:
Yes please, ideally we can then put that issue here so we know when we can test this again.

Reply:
I've added the issue link to the comment